api/SVGZoomAndPan.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGZoomAndPan": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGZoomAndPan",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
All 3 versions of Chrome should be `true`. https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/svg/svg_zoom_and_pan.idl